src/main/java/com/list/todo/entity/TodoListStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.list.todo.entity;
+
+public enum TodoListStatus {
+    Active,
```

Comment:
please follow java naming conventions `Because they are constants, the names of an enum type's fields are in uppercase letters.`
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html